index.ftd

```diff
@@ -115,33 +147,41 @@ Body text here in h3
 
 
 
--- h0: Heading Hero
-
-Body text is here in h0
-
-
-
--- h1: Heading Large
-
-Body text is here in h1
-
-
 
--- h2: Heading Medium
 
-Body text is here in h2
 
 
 
--- h3: Heading Small
+-- component h1:
+caption title:
+optional body body:
 
-Body text is here in h3
+-- ftd.column:
+width: fill-container
+region: h1
+;;append-at: inner -- TODO FTD 0.3
```

Comment:
`append-at` is never going to come in `FTD 0.3`. Remove this line from everywhere